Portfolio/index.html

```diff
@@ -1,16 +1,65 @@
-<!DOCTYPE html>
-<html>
-  <head>
-    <title>Portfolio</title>
-    <meta charset="UTF-8" />
-    <link rel="stylesheet" href="index.css" />
-  </head>
-
-  <body style="margin: 0; padding: 0">
-    <div top="0">
-      <a href="../index.html">Go Back</a>
-    </div>
-    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
-    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
-  </body>
-</html>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Portfolio</title>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="index.css" />
+  </head>
+
+  <body style="margin: 10px; padding: 0">
+    <div top="0">
+      <a href="../index.html">Go Back</a>
+    </div>
+    <!-- Write your code below for PORTFOLIO and DO NOT TOUCH the code above-->
+    <!-- to import image from public folder, do it like this= <img src="../public/css.icon.png" alt="CSS image"> -->
+    <h1>Pratik Shrivastava</h1>
+    <ul>
+        <li>ABOUT</li><br>
+        <li>SKILLS</li><br>
+        <li>PROJECTS</li><br>
+        <li>REVIEWS</li><br>
+    </ul>
+    <button>CONTACT ME</button>
+    <h2>Pratik Shrivastava</h2>
+    <img src="https://avatars.githubusercontent.com/u/108012653?v=4" alt="CSS image">
+    <h1>Front End</h1>
+    <h1>Developer</h1>
+    <h2>GET TO KNOW ME</h2>
+    <p> I'm a seasoned front-end web Developer with a passion for crafting<br>
+        exceptional digital experiences. With over 2 years of hands-on experience, <br>
+        I bring a deep proficiency in HTML, CSS, JavaScript, Typescript, and a<br>
+        keen mastery of modern frameworks like React.js, React Native, Sveite.js.<br>
+        I specialize in building responsive, interactive and visually appealing user<br>
+        interfaces. Additionally, my proficiency in Firebase ensures secure and<br>
+        efficient backend integration, making me a versatile and dynamic<br>

```

Comment:
The closing tag for <h3> on line 50 does not match the opening tag. Update it to </h3>.